### DISCUSSION TITLE
Query result panel arrange

### DISCUSSION THREAD

**Discussion by zmagyar77** [dbeaver/dbeaver#22573] [General]
Is it possible to arrange the result panels of queries in a same query editor, horizontally?
I mean, like this in case of MS Sql studio:
![image](https://github.com/dbeaver/dbeaver/assets/22811037/32df3011-79f6-4c17-8dd4-f43cac435e31)
So I want to see all query results at a glance!

Thanks!

**Comment by E1izabeth** [comment]
You can enable this mode here
![image](https://github.com/dbeaver/dbeaver/assets/28875055/aa67ca28-2ab6-4603-96e4-841aa492b51b)

Also it's possible to bring this mode switcher to the left-side toolbar to have it here
![image](https://github.com/dbeaver/dbeaver/assets/28875055/13cc9f51-e9d1-4a60-8ea7-a12cc6edcb93)
 
![image](https://github.com/dbeaver/dbeaver/assets/28875055/d552be5e-140d-4ba2-b5cf-52590726a99e)

**Reply by zmagyar77** [reply]
Thank you for your quick answer. I Tried this toggle, but the result tabs are visible next to each other, not underneath each other. How can I commit the select, or set the behavior of these tabs, to see their all contents of results at same time? Not only one by one.

**Reply by E1izabeth** [reply]
It works when you execute queries in a script mode, not one-by-one.

**Reply by E1izabeth** [reply]
This way
![image](https://github.com/dbeaver/dbeaver/assets/28875055/9d143eb1-b12b-4012-b974-13c61c2db757)
